demos/demo_runner/iot_demo_freertos.c

```diff
@@ -346,6 +359,13 @@ void runDemoTask( void * pArgument )
         IotLogError( "Failed to initialize the demo. exiting..." );
     }
 
+    #ifdef democonfigMEMORY_ANALYSIS
+        xAfterDemoHeapSize = demoMEMORY_ANALYSIS_MIN_EVER_HEAP_SIZE();
+        IotLogInfo( "Demo Memory Analysis Heap Total: %u Before: %u After: %u \r\n", configTOTAL_HEAP_SIZE, xBeforeDemoHeapSize, xAfterDemoHeapSize );
+        xAfterDemoTaskWaterMark = demoMEMORY_ANALYSIS_STACK_WATERMARK( NULL );
+        IotLogInfo( "Demo Memory Analysis Stack Total: %u Watermark Before: %u After: %u \r\n", xMemoryAnalysisStackSize, xBeforeDemoTaskWaterMark, xAfterDemoTaskWaterMark );
```

Comment:
could we directly access 'democonfigDEMO_STACKSIZE' here instead of 'xMemoryAnalysisStackSize'

Reply:
I tried, but it gets redefined in demo_runner.h. So democonfigDEMO_STACKSIZE always had the default stack size as it's value. Open to suggestions

Reply:
Ok, that makes sense, Can we rename 'xMemoryAnalysisStackSize' to 'xDemoStackSize' to be more clear.

Reply:
Yes I will do that

Reply:
Done.